test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -241,7 +257,7 @@ export default class BaseFormObject {
 
   getAlertBoxText() {
     if (userVariables.v3) {
-      return within(this.el).queryAllByRole('alert').nth(0).innerText;
+      return this.getAlertBox().innerText;
```

Comment:
There seems to be a reason the original function was made that way, it seems like there are additional elements on the page (that uses this function) that has the role of `alert`. You may need to use the `getAllAlertBoxes` function you created earlier.

Reply:
yeah I believe it was for when there was alert box errors aswell as field errors at the same time since they both have roles of 'alert'.  The `getAlertBoxByIndex` function uses the `getAllAlertBoxes` function inside it so the logic should be the same.